src/index.js

```diff
@@ -37,6 +37,22 @@ function cachedIsFile (file, cb) {
 	isFileCache[file].then(contents => cb(null, contents), cb);
 }
 
+let isDirCache = {};
```

Comment:
I know we also do it for files this way (and this should also be changed), but if we want to use an object for cache storage, we should initialize it with `Object.create(null)`. Two reasons:
- avoid conflicts with builtin `Object` properties
- the V8 compiler will automatically use its "dictionary mode" for such an object, avoiding unnecessary hidden class operations and providing better performance.

However after talks with the V8 team, I would actually recommend switching to `Map` for all our caching. As our baseline compatibility is Node 6, this is not an issue from this side.

Citing the V8 team: "Using `Map` expresses an intent that enables the compiler to do much more targeted optimizations. Even though syntactically we are writing function calls, they will be quite probably be inlined by the compiler and it should be even faster than using an object in recent V8 versions"

Reply:
Also you should check the linter which forces you to use `const` here.

Reply:
Ahh this is because I accidentally a line; we're supposed to clear the cache out in `generateBundle`